content/guides/production-build.md

```diff
@@ -188,7 +188,10 @@ module.exports = function() {
                 use: 'file-loader'
             }, {
                 test: /\.(woff|woff2|eot|ttf|svg)$/,
-                use: 'url-loader?limit=100000'
+                use: 'url-loader',
```

Comment:
`use` and `options` won't work (forbidden). This has to be literally

```
use: {
  loader: 'url-loader',
  options: {
    limit: 100000
  )
)
```

After that it's valid. 👍